.circleci/config.yml

```diff
@@ -335,6 +335,70 @@ jobs:
             git checkout -b release-client-v${VERSION}
             git push -u origin release-client-v${VERSION}
 
+  notify-stuck-stage-nodes-job:
+    resource_class: small
+    docker:
+      - image: cimg/base:2023.01
+    steps:
+      - run:
+          name: Alert Slack of stuck stage nodes
+          command: |
+            handle_error() {
+              # Construct failure Slack message
+              failure_content="{ \"blocks\": ["
+              failure_content+="{ \"type\": \"section\", \"text\": { \"type\": \"plain_text\", \"text\": \"Encountered error while checking for stuck staging nodes\n\" } }"
+              failure_content+="]}"
+              echo "Sending error message to Slack: $failure_content"
+
+              # Send Slack failure message
+              curl -f -X POST -H 'Content-type: application/json' \
+                --data "$failure_content" \
+                $SLACK_DAILY_DEPLOY_WEBHOOK
+            }
+
+            (
+              set -e
+
+              # Fetch the latest version from the GitHub repository (assume Content and Discovery have the same latest versions)
+              versionUrl="https://raw.githubusercontent.com/AudiusProject/audius-protocol/main/packages/discovery-provider/.version.json"
+              VERSION=$(curl -s "$versionUrl" | jq -r '.version')
+
+              if [ -z "$VERSION" ]; then
+                echo "Failed to fetch version data"
+                exit 1
+              fi
+
+              contentEndpoints=("https://creatornode5.staging.audius.co" "https://creatornode6.staging.audius.co" "https://creatornode7.staging.audius.co" "https://creatornode8.staging.audius.co" "https://creatornode9.staging.audius.co" "https://creatornode10.staging.audius.co" "https://creatornode11.staging.audius.co" "https://creatornode12.staging.audius.co")
```

Comment:
Prefer not to hard-code these, but won't block

Reply:
changed and retested 👍 now it only hardcodes as a fallback in case fetching from api.staging.audius.co fails